packages/lib-classifier/src/store/WorkflowStepStore.spec.js

```diff
@@ -185,71 +187,50 @@ describe.only('Model > WorkflowStepStore', function () {
         }
       })
 
-      classification = ClassificationFactory.build({
-        annotations: [
-          { task: 'T1', value: 0 },
-          { task: 'T2', value: [0, 2] }
-        ]
-      })
-
-      flaggedClassification = ClassificationFactory.build({
-        annotations: [
-          { task: 'T1', value: 0 },
-          { task: 'T2', value: [0, 2] }
-        ],
-        metadata: {
-          subject_flagged: true
-        }
-      })
+      subject = SubjectFactory.build()
     })
 
     after(function () {
       ROOT_STORE_INSTANCE = null
+      subject = null
       WORKFLOW = null
     })
 
-    it('should return false if there is not an active workflow', function () {
+    function setupDoneAndTalkTest ({ workflow, createClassification }) {
```

Comment:
Maybe rename `createClassification` to `withClassification`. I thought it was a reference to the function of the same name.